### DISCUSSION TITLE
Hierarchy display in a expand and collapse I need to show metadata against to each row item in tabular with search

### DISCUSSION THREAD

**Discussion by a658664** [atoti/atoti#772] [Show and tell]
Hello, I have used Atoti to Visualization of data in parent and child relationship, I need to show details of metadata in Tabular or some other format against to each levels in hierarchy on click of expand and collapse format for Parent & child row. 
The Search filter should search the data even from metadata , I am unable to display metadata & search  against to that

`import atoti 

session = atoti.Session()

events_ds = session.read_csv(
    "C:\\app\\folder-hierv2.csv",
    keys=["ID"],
)
events_ds.head()
cube = session.create_cube(events_ds)
cube.hierarchies["DISPLAY_DATA"] = [
    cube.levels["PARENT"],
    cube.levels["CHILD"]
    
]

session.visualize()

session.link()`

Atoti Visualization:
![image](https://user-images.githubusercontent.com/127825929/224917069-01bdf96e-7863-4b5e-b937-9cdca44e0a97.png)

Sample data:
![image](https://user-images.githubusercontent.com/127825929/224918015-a74e4b60-ef91-4c52-9eec-b108ada45fc0.png)

**Comment by tibdex** [comment]
I'm sorry but I don't understand what you want. Can you try to reformulate?

**Reply by a658664** [reply]
Hello @tibdex  Thanks for response, I mean I have to Show details last 3 columns(db,schema,objects)against to each hierarchy level (Parent & Child columns).
If I do an custom search It should display records either Parent to child or from Objects to parents

**Reply by a658664** [reply]
@tibdex  Could you please suggest, appreciate your response
